src/components/FilmCard.js

```diff
@@ -3,17 +3,10 @@ import listAction from './listAction.json';
 import '../styles/FilmCard.css';
 
 class FilmCard extends React.Component {
-  constructor (props) {
-    super(props);
-    this.state = {
-      index: 0
-    };
-  }
-
   render () {
-    const posterPath = listAction.results[this.state.index].poster_path;
-    const posterPath2 = listAction.results[this.state.index + 1].poster_path;
-    const posterPath3 = listAction.results[this.state.index + 2].poster_path;
+    const posterPath = listAction.results[this.props.index].poster_path;
+    const posterPath2 = listAction.results[this.props.index + 1].poster_path;
+    const posterPath3 = listAction.results[this.props.index + 2].poster_path;
```

Comment:
Attention : quand on arrivera en bout de liste, listAction.results[this.props.index + 1] sera undefined